lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -10,9 +10,10 @@ def name
       # need: params['percentage']
       def is_enabled?(params = {}, context = nil)
         return false unless params.is_a?(Hash)
-        return false unless context.instance_of?(Unleash::Context)
 
         stickiness = params.fetch('stickiness', 'default')
+        return false if context_invalid?(stickiness, context)
```

Comment:
This is very easy to read!

But, Is the intention to have `is_enabled?` always return `false` when (one of):
- context is not `Unleash::Context` (ok!)
- stickiness is `default` (ok?)
- stickiness is `random` (???)

Reply:
if stickiness is default or random continue evaluating the rollout

if stickiness is something else, require to have unleash::context

Reply:
I understand that's the desired behavior. Any chance you can add separate unit tests for these scenarios?

Reply:
ok